samples/Mvc.Server/Controllers/AuthorizationController.cs

```diff
@@ -52,7 +53,9 @@ public class AuthorizationController : Controller {
 
             return View(new AuthorizeViewModel {
                 ApplicationName = application.DisplayName,
-                Parameters = request.Parameters,
+                Parameters = request.ToDictionary(
```

Comment:
@ilmax I'm considering simply flowing the `request_id` instead of the entire OIDC payload, which would make views and view models a bit simpler and would prevent devs from messing with the OIDC parameters. Any objection?